dsgrid/config/dimension_associations.py

```diff
@@ -42,7 +42,7 @@ def load(cls, path: Path, association_files):
         associations = {}
         for association_file in association_files:
             filename = path / association_file
-            table = read_dataframe(filename, cache=True)
+            table = read_dataframe(filename, cache=True, read_with_spark=False)
```

Comment:
Should read_with_spark be an option for the load method? (I.e., is it always always going to be the case that we want read_with_spark to be False here?)

Reply:
I think we always want it to be False. On local systems it really doesn't matter. On EC2 systems it will fail if we try to read with Spark.